ui/src/pages/frontsite/PatronProfile/PatronCurrentDocumentRequests/PatronCurrentDocumentRequests.js

```diff
@@ -50,23 +52,62 @@ export default class PatronCurrentDocumentRequests extends Component {
 
   renderLoader = props => {
     return (
-      <>
-        <Item.Group>
-          <ILSItemPlaceholder fluid {...props} />
-        </Item.Group>
-      </>
+      <Item.Group>
+        <ILSItemPlaceholder fluid {...props} />
+      </Item.Group>
     );
   };
 
   renderNoResults = () => {
     return (
       <NoResultsMessage
-        messageHeader={'No loan requests'}
-        messageContent={'Currently you do not have any loan requests'}
+        messageHeader={'No literature requests'}
```

Comment:
I am sorry if I stress on this: if I am the user, and I never did a request, I go to my profile, not sure I understand what is that.

What do you think of:
```
**Requests for new literature**
You did not create any request for literature not available in the library catalog yet.
```
And I would add a button:
```
Create a new request
```

Reply:
The message is misleading indeed, I am not sure about the button though, maybe @kprzerwa has a stronger opinion 🥇

Reply:
I would add a link to search first, not a button to create a new request, to invite the user to search, instead of creating the requests for something which might exist